GeoHealthCheck/config_main.py

```diff
@@ -47,6 +47,7 @@
 GHC_NOTIFICATIONS_EMAIL = ['you2@example.com']
 GHC_SITE_TITLE = 'GeoHealthCheck Demonstration'
 GHC_SITE_URL = 'http://host'
+GHC_METADATA_CACHE_SECS = 7200
```

Comment:
Yes, but oops, `GHC_METADATA_CACHE_SECS` is not used yet....should be used here:
https://github.com/geopython/GeoHealthCheck/blob/dev/GeoHealthCheck/probe.py#L120